a10_octavia/common/config_options.py

```diff
@@ -271,9 +271,18 @@
                default=900,
                help=_('Timeout for waiting when vThunder instance is busy. '
                       '(0 for no timeout')),
-    cfg.IntOpt('max_db_timeout',
-               default=60,
-               help=_('Database Timeout for creating a Database entry'))
+    cfg.IntOpt('retry_attempts',
+               default=15,
+               help=_('Retry attempts for Database Entry')),
+    cfg.IntOpt('retry_initial_delay',
+               default=1,
+               help=_('Retry initial delay for Database Entry')),
+    cfg.IntOpt('retry_bakcoff',
+               default=1,
+               help=_('Retry Backoff for Database Entry')),
```

Comment:
Can we change to "Retry delay backoff for database fetching"

Reply:
Fixed